app/assets/javascripts/discourse/app/controllers/user-summary.js

```diff
@@ -10,6 +10,7 @@ const MAX_BADGES = 6;
 export default Controller.extend({
   userController: inject("user"),
   user: alias("userController.model"),
+  showStats: readOnly("model.can_see_summary_stats"),
```

Comment:
I would not even bother making this property, myself. You can have the template check `model.can_see_summary_stats` directly.